src/components/ProductVersionReleasesList/ProductVersionReleasesList.tsx

```diff
@@ -50,7 +50,13 @@ export const ProductVersionReleasesList = ({
             <Thead>
               <Tr>
                 <Th width={25}>{productReleaseEntityAttributes.version.title}</Th>
-                <Th width={20}>{productReleaseEntityAttributes.releaseDate.title}</Th>
+                <Th
+                  info={{
```

Comment:
@DnsZhou already started to use our tooltip implementation, see https://github.com/project-ncl/pnc-web-ui-react/blob/main/src/components/ScmRepositoriesList/ScmRepositoriesList.tsx#L80

We should be using the same also here. The advantage is that the same tooltip design (icon, tooltip itself) is used also on other pages like detail page.

Apply the same on all other places.

Reply:
Anyway, thanks for applying given change also on this place!